react/week2/todolist/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import logo from './logo.svg';
+import './App.css';
+
+let tasks = ["submit hyf homework", 
+            "go for a run", 
+            "watach movie"];
+
+            
+function TodoTimer(props){
+  const [count, setCount] = useState(0);
+  setTimeout(() => setCount(count + 1), 1000);
+  return <p>You have used this app for {count} seconds</p>;
+};
+
+function Todo(props){
+  
+  const [checked, updateCheck] = useState(false)
+  const taskDone = checked ? "done" : ""
+
+  return (
+    
+    <div className={taskDone}>
+      <input type="checkbox" onClick={() => {updateCheck(!checked)}}></input>
+      <label>{props.name}</label>
+    </div>
+  
+  )
+}
+
+function App() {
+
+  const [todos, updateTodos] = useState(tasks)
+
+
+  const onAdd = () => {
+    console.log ("adding new todo")
+    const newTodo  = document.getElementById("newTodo").value;
+    let newTodoList = todos.concat(newTodo)
+    updateTodos(newTodoList)
+    console.log(todos)
+  }
+
+  const onDelete = (index) => {
+    console.log("delete clicked" + index)
+    console.log(todos)
+    let newTodoList = todos.filter((data, idx) => idx !== index); 
+    updateTodos(newTodoList)
+    console.log(todos)
```

Comment:
```suggestion
  const onDelete = (index) => {
    let newTodoList = todos.filter((data, idx) => idx !== index); 
    updateTodos(newTodoList)
```

Reply:
I removed all console and also changed delete functionality using id.